apps/e2ee-server/src/utils/bufferUtils.ts

```
@@ -0,0 +1,91 @@
+
+import hexUtils from './hexUtils';
+
+function toBuffer(
+  data: Buffer | ArrayBuffer | Uint8Array | string,
+  // encoding of string data
+  encoding: BufferEncoding = 'hex',
+): Buffer {
+  if (isString(data)) {
+    if (encoding === 'hex') {
+      // eslint-disable-next-line no-param-reassign
+      data = hexUtils.stripHexPrefix(data);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)